Main_for_ODBS.py

```diff
@@ -0,0 +1,150 @@
+    with a length lower than this value.
+    :param quality_threshold: int. All reads with quality below the value will be discarded.
+    :return: dict.
+    """
+    filtered_seqs = {}
+    fastq_file = read_fastq_file.read_fastq(input_path)
+    for name, (seq, comment, qual) in fastq_file.items():
+        if FASTQ_tools.filter_gc(seq, gc_bounds) and FASTQ_tools.filter_len(seq, length_bounds) \
+                and FASTQ_tools.filter_qual(qual, quality_threshold):
+            filtered_seqs[name] = (seq, comment, qual)
+    write_output.save_filtered_fastq(filtered_seqs, input_path, output_filename)
+
+
+def run_dna_rna_tools(*sequences):
+    """
+    This run_dna_rna_tools function takes as input an arbitrary number of arguments with DNA or/and RNA sequences,
+    as well as the name of the procedure to be executed (which is the last argument, str).
+    Possible procedures performed by the program:
+    - transcribe — print the transcribed sequence. If an RNA sequence is supplied as input,
+    then the program is interrupted and generates an error.
+    - reverse — print the reversed sequence
+    - complement — print the complementary sequence
+    - reverse_complement — print the reverse complementary sequence
+    - gc_content—calculation of GC content in sequence,
+    :param sequences: str, can be more than 1 seq, only 1 procedure.
+    :return: str or list. If one sequence is given as input, a string with the result is returned.
+    If several sequences are supplied, a list of strings is returned.
+    """
+    procedure = sequences[-1]  # Unpack procedure
+    sequences = sequences[:-1]
+
+    results = []
+    dna_nucleotide_alphabet = {'A', 'T', 'G', 'C', 'a', 't', 'g', 'c'}
+    rna_nucleotide_alphabet = {'A', 'U', 'G', 'C', 'a', 'u', 'g', 'c'}
+    for sequence in sequences:
+        for nucleotide in sequence:
+            if nucleotide in dna_nucleotide_alphabet:
+                continue
+            elif nucleotide in rna_nucleotide_alphabet:
+                continue
+            else:
+                raise ValueError('this is not a nucleotide sequence')
+        if procedure == 'transcribe':
+            result = dna_rna_tools.transcribe(sequence)
+        elif procedure == 'reverse':
+            result = dna_rna_tools.reverse(sequence)
+        elif procedure == 'reverse_complement':
+            result = dna_rna_tools.reverse_complement(sequence)
+        elif procedure == 'complement':
+            result = dna_rna_tools.complement(sequence)
+        elif procedure == 'gc_content':
+            result = dna_rna_tools.gc_content(sequence)
+        else:
+            result = "Invalid procedure"
+        results.append(result)
+
+    if len(results) == 1:
+        return results[0]
+    else:
+        return results
+
+
+def run_aminoacid_seq(sequence: str, function: str = 'summary', record_type: int = 1, percent: bool = False):
```

Comment:
```suggestion
def run_aminoacid_tools(sequence: str, function: str = 'summary', record_type: int = 1, percent: bool = False):
```
Наверное так. Ты же тулы запускаешь,  а не аминокислотную последовательность:)